app/src/main/res/values/strings.xml

```diff
@@ -11,6 +11,7 @@
     <string name="open_in_browser">Open in browser</string>
     <string name="open_in_popup_mode">Open in popup mode</string>
     <string name="share">Share</string>
+    <string name="share_stream">Share link to stream file</string>
```

Comment:
I don't like this string much. What about "Share stream file link" or "Share stream link"? @B0pol @wb9688 @opusforlife2 do you have any other idea?

Reply:
My vote goes to Share Video Link. I'm assuming this doesn't work for background and popup.

Reply:
I picked this name to be consistent with `controls_download_desc` (`Download stream file`). Looking into it, that string is only used for accessibility so it may not even make sense to follow the same naming scheme.

In any case, I agree that the string is not ideal. Personally, I would go with `Share stream link`.

Reply:
@opusforlife2 It works for audio streams too (e.g. soundcloud)

Reply:
What about youtube audio stream?

Reply:
I haven't checked against youtube audio streams. If you can provide such a link I could try it.

I will also check it with YouTube livestreams (haven't checked those yet).

Reply:
No idea how Newpipe gets those. One of the devs could help you there.

I'm thinking that there should be some way to choose between sharing the video stream or audio stream to send to the external player.

Reply:
I think @opusforlife2 is talking about YouTube audio streams (those played with the background player). Maybe two buttons should be added, one to share the video stream and another to share the audio stream? I am not sure though, since it would mean adding two buttons used only by a small group of people, so do not implement this yet, I'll ask for some opinions internally.
Btw, I am ok with `Share stream link`

Reply:
For now, let's only add the ability to share the video stream link (and the audio one for SoundCloud, i.e. the current behaviour), to be consistent with the fact that there is no way to open an audio stream link with the external player. So keep the current changes and just change the value of the string to `Share stream link`

Reply:
Ah got it. I somehow thought YouTube had audio-only streams. In the meantime, I tried playing some YouTube live streams, and the link is shared correctly but it won't play in VLC. I guess that's a problem with the external player rather than NewPipe itself.

As for choosing between audio-only vs. video, I guess it could be made into a submenu, similar to the one used to pick the stream quality in the action bar. IMO it's overkill unless there is demand for this feature. I don't mind implementing that if/when it's needed.

The string should be updated now.

Reply:
what about share stream URL?

Reply:
I also thought about "url", but then I realized it is more of a technical word, and "link" is more used by the average non-tech-savy person. For example, Google uses "link" in Meet invites, and other people I know use that term, too (this could be biased by the fact that it is used in the italian language, though). So I'd keep "link".

Reply:
You are right. Though URL is often used in newpipe strings.

Reply:
> No idea how Newpipe gets those. One of the devs could help you there.
> 
> I'm thinking that there should be some way to choose between sharing the video stream or audio stream to send to the external player.

I think @ AudricV can shed some light on that